charts/hoprd-operator/templates/configmap-scripts.yaml

```diff
@@ -58,4 +58,4 @@ data:
     set -Eeuo pipefail
     export PATH=${PATH}:/root/.foundry/bin/
     export RUST_BACKTRACE=1
-    hopli faucet --environment-name ${HOPRD_ENVIRONMENT} --use-local-identities --identity-directory /app/hoprd-identity/ --address ${HOPRD_ADDRESS} --contracts-root /root/contracts --hopr-amount 10 --native-amount 1
\ No newline at end of file
+    hopli faucet --environment-name ${HOPRD_ENVIRONMENT} --use-local-identities --identity-directory /app/hoprd-identity/ --address ${HOPRD_ADDRESS} --contracts-root /root/contracts --hopr-amount 10 --native-amount 0.1
```

Comment:
It already needed to be changed, so it cloud be made into an env variable, or not? Different users might need different amount of xDAI.

Reply:
That's not the idea. The idea is to bring up nodes with the least funds. The reason to reduce the funding is because it was too much, 1$, per node. Now is less. Nevertheless, now that I see the value it should be 0,01 which is the minimum.